lib/BBTools/utils/commandbuilder.py

```diff
@@ -14,6 +14,7 @@ def build_options(params, available_params):
             continue
         elif param not in available_params:
             #raise ValueError("Can't parse unknown parameter {}".format(param))
+            print("WARNING build_options(): "+"Can't parse unknown parameter {}".format(param))
```

Comment:
f-strings again are helpful.

Reply:
see above about my concerns about porting to python3 as part of this particular PR

Reply:
(f-strings does look like nicer syntax, BTW)